### PR TITLE
Remove restriction on mathml deprecated features

### LINKED ISSUE #2118: Does the restriction on Deprecated MathML cause retroactive content breakage?
In the section on [MathML restrictions](https://w3c.github.io/epub-specs/epub33/core/#sec-xhtml-mathml)

> EPUB Creators MUST NOT include elements and attributes marked as deprecated in [[MATHML3](https://w3c.github.io/epub-specs/epub33/core/#bib-mathml3)] within MathML markup in XHTML Content Documents.

Does this mean, in this era of living standards, that as MathML adds items to the deprecated list existing EPUB content is retroactively made non-conforming?

### PR DESCRIPTION
**Pull request by mattgarrish**
I've also added the general caution from #2118 to the top of the relationships to other specifications section and removed the individual caution boxes from the HTML and SVG subsections.

I've also added a brief "relationship to mathml" section to note that we only directly support presentation markup.

Fixes #2118

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/pull/2137.html" title="Last updated on Mar 31, 2022, 11:14 AM UTC (de3addf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/2137/e05af7d...de3addf.html" title="Last updated on Mar 31, 2022, 11:14 AM UTC (de3addf)">Diff</a>

### CODE REVIEW

File: epub33/core/index.html
Comment (iherman) [inline epub33/core/index.html]: The sentence sounds strange. It could be read as if those other standards are changed by EPUB...

Maybe saying "references to others"? Or say that those standards are subject to change rarely by a Working Group, typically only once in a few years.

Comment (mattgarrish) [reply epub33/core/index.html]: Ya, I've tried to clear this up in the latest commit and also pare down the caution a bit to make it more succinct.

See if it works better now.

**Review by iherman — APPROVED** [review]

**Comment by mattgarrish** [comment]
Since we're removing the restriction, probably another to get a WG resolution on before merging.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2022-03-31-epub#section2) on 2022-03-31

List of resolutions:

- ***[Resolution No. 2](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2022-03-31-epub#resolution2):  merge PR 2137, close issue 2118.***

<details><summary><i>View the transcript</i></summary>

### 2. Remove restriction on mathml deprecated features (pr epub-specs#2137)

_See github pull request [epub-specs#2137](https://github.com/w3c/epub-specs/pull/2137)._




_See github issue [epub-specs#2118](https://github.com/w3c/epub-specs/issues/2118)._




**Matt Garrish:** duga had opened this initially, and we went back and forth.  
… i think his concern is that we're imposing a restriction on MathML's deprecated features that MathML itself doesn't impose on its features.  
… they are deprecated, but you're still allowed to have them in MathML.  
… we are imposing on their spec.  
… he's asked that we not restrict these deprecated features.  
… I agree, and I don't think us putting a ban on MathML features moves the needle on MathML rendering.  
… math rendering is made for web browsers, and we just use what they do.  
… if getting rid of these restrictions smooths over a spec weirdness, that is fine.  
… people don't hand write MathML anyway.  
… so this PR takes out the MUST NOT that was there before.  

**Dave Cramer:** this is in keeping with changes we've been making incrementally over the last 5-10 years.  
… initially we were intervening with the goal of making it easier for people who were trying to write RS from scratch.  
… to allow them to skip support for complicated features.  
… this PR is in keeping with our take on epub as a container for these other technologies.  
… we leave it up to these other specs.  

**Matt Garrish:** I don't think these restrictions work anymore, but they were important at the time.  

**Dave Cramer:** so I would propose we merge this.  

> *Murata Makoto:* +1.

> **Proposed resolution: merge PR 2137, close 2118.** *(Dave Cramer)*

> *Dave Cramer:* +1.

> *Shinya Takami (高見真也):* +1.

> *Masakazu Kitahara:* +1.

> *Matt Garrish:* +1.

> *Toshiaki Koike:* +1.

> *Matthew Chan:* +1.

> ***Resolution #2: merge PR 2137, close issue 2118.***

</details>

**Pull request merged**
